packages/hooks/src/use-clipboard.ts

```diff
@@ -7,29 +7,37 @@ import copy from "copy-to-clipboard"
  * @param text the text or value to copy
  * @param timeout delay (in ms) to switch back to initial state once copied.
  */
-export function useClipboard(text: string, timeout = 1500) {
+export function useClipboard(
+  text: string,
+  timeout:
```

Comment:
What do you think about renaming `timeout` to `optionsOrTimeout, as it could be either an options object or timeout number`?

Reply:
Agree!